docs/handbook/tutorial.rst

```diff
@@ -522,6 +493,42 @@ Reading from a tar archive
     fp = TarIO.TarIO("Tests/images/hopper.tar", "hopper.jpg")
     im = Image.open(fp)
 
+
+Batch processing
+^^^^^^^^^^^^^^^^
+
+Operations can be applied to multiple image files. For example, all PNG images
+in the current directory can be saved as JPEGs at reduced quality.
+
+::
+
+    import os
+    from PIL import Image
+
+
+    def compress_image(source_path, dest_path):
+        with Image.open(source_path) as img:
+            if img.mode != "RGB":
+                img = img.convert("RGB")
+            img.save(dest_path, "JPEG", optimize=True, quality=80)
+
+
+    paths = [path for path in os.listdir(".") if path.endsWith(".png")]
```

Comment:
Not essential to change, but I'd `import glob` and use something like:
```suggestion
    paths = glob.glob(".png")
```

https://docs.python.org/3/library/glob.html